site/docs/language/grammar.mdx

```diff
@@ -72,77 +72,164 @@ As a general rule, we don’t use bold text for emphasis or to draw a user’s a
 
 ### Capitalization
 
-We use various forms of capitalization at Culture Amp. Here are the common use cases:
+Culture Amp’s general product style is sentence case. This means the first word in a sentence and proper nouns are capitalized (like names of people, places and businesses). All other words are lowercase. 
 
+We consider brand names, product names, and a few selected features to be proper nouns. Therefore, they have all words in the name capitalized. Here’s a list of agreed on proper nouns. 
 
+Agreed proper nouns (which are always capitalized):
```

Comment:
Nitpicky, but I feel like this is a bit repetitive when combined with the sentence above. We could condense these two paragraphs a bit more:

> We consider brand names, product names, and a few selected features to be proper nouns. Therefore, they have all words in the name capitalized. Here’s a list of agreed on proper nouns (which are always capitalized).

Reply:
Sounds good 👍